parameterized-utils.cabal

```diff
@@ -69,6 +69,7 @@ library
   exposed-modules:
     Data.Parameterized
     Data.Parameterized.All
+    Data.Parameterized.Axiom
```

Comment:
I went ahead and exported this, since it's conceivable that a user might want to use `axiom` to construct proofs that aren't provided by `parameterized-utils`. I don't have a strong opinion on this, however, and I would be find with making it an internal module if others feel differently.

Reply:
Thanks for doing this! I've wanted something like `axiom` before just to make it clear what was going on, so I like exporting it to provide a nice common name. It seems to be guarded appropriately by scary comments.

Reply:
Would unsafeAxiom be a better name?

Reply:
Sounds good to me. I've pushed a change renaming `axiom`/`haxiom` to `unsafeAxiom`/`unsafeHeteroAxiom`.